sr/comp/cli/show_league_table.py

```diff
@@ -19,18 +20,27 @@ def command(settings):
     for team in comp.teams.values():
         scores = comp.scores.league.teams[team.tla]
         league_pos = comp.scores.league.positions[team.tla]
-        league_table.append((
+        league_table.append([
             league_pos,
             scores.league_points,
             scores.game_points,
             team.tla,
             team.name,
-        ))
+        ])
 
     # sort rows
     sort_col, sort_direction = SORT_TYPES[settings.sort]
     league_table.sort(key=lambda row: row[sort_col], reverse=sort_direction)
 
+    # calculate ties
+    tie_count = Counter(rank for rank, _, _, _, _ in league_table)
```

Comment:
Feels like this might be simpler if you pulled the data from `comp.scores.league.positions.values()` instead?